numpy/random/__init__.pyi

```diff
@@ -8,57 +8,58 @@ from numpy.random._philox import Philox as Philox
 from numpy.random._sfc64 import SFC64 as SFC64
 from numpy.random.bit_generator import BitGenerator as BitGenerator
 from numpy.random.bit_generator import SeedSequence as SeedSequence
+from numpy.random.mtrand import (
```

Comment:
Does importing here work differently than regular python that we need to do "from x import y as y" and not "from x import y"? If so, it would be nice to get a comment explaining this for the next time I ask the same question

Reply:
Taking @BvB93 's word, yes.  If you want a symbol reexported, then you need to import X as X.

Reply:
Per PEP [484](https://www.python.org/dev/peps/pep-0484/):

> Modules and variables imported into the stub are not considered exported from the stub unless the import uses the `import ... as ...` form or the equivalent `from ... import ... as ...` form. (UPDATE: To clarify, the intention here is that only names imported using the form X as X will be exported, i.e. the name before and after as must be the same.)

Reply:
Thanks. I probably asked this last time I looked at this code.